apscheduler/util.py

```
@@ -1,10 +1,11 @@
 """This module contains several handy functions primarily meant for internal use."""
+import sys
+from datetime import datetime, timedelta, tzinfo
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import

(F401)